tools/regenerateAll.bat

```diff
@@ -0,0 +1,72 @@
+@echo off
+set repoUser=olydis
+set repoBranch=batches

```

Comment:
this file is rather temporary and for my own testing - @shahabhijeet not sure whether it also makes sense otherwise. If so, I'll if course adjust this!